deploy/monai-deploy-express/init.sh

```diff
@@ -18,6 +18,13 @@ RUNDIR=$PWD/.md
 echo "Initializing directories..."
 [ -d $RUNDIR ] && echo "Removing existin $RUNDIR" && sudo rm -r $RUNDIR
 mkdir -p $RUNDIR/esdata/ && echo "Created $RUNDIR/"
+mkdir -p $RUNDIR/minio/ && echo "Created $RUNDIR/minio/"
+mkdir -p $RUNDIR/rabbitmq/ && echo "Created $RUNDIR/rabbitmq/"
+mkdir -p $RUNDIR/orthanc/ && echo "Created $RUNDIR/orthanc/"
+mkdir -p $RUNDIR/mongodb/ && echo "Created $RUNDIR/mongodb/"
+mkdir -p $RUNDIR/mdwm/ && echo "Created $RUNDIR/mdwm/"
+mkdir -p $RUNDIR/mdtm/ && echo "Created $RUNDIR/mdtm/"
+mkdir -p $RUNDIR/mdig/ && echo "Created $RUNDIR/mdig/"
 sudo chown 1000:1000 -R $RUNDIR/esdata && echo "Permission updated"
```

Comment:
Container image user 1000:1000 is the default user for MAP built with MD Packager v0.6, though the user of Packager can set it to others. Are we only support user 1000? (I'll see in the other files if the user and file permissions are adjusted per WF def.)

Reply:
This is actually unrelated, but the value should be `$USER`